src/libstd/sys/redox/os.rs

```diff
@@ -209,3 +209,7 @@ pub fn exit(code: i32) -> ! {
     let _ = syscall::exit(code as usize);
     unreachable!();
 }
+
+pub fn getpid() -> u32 {
+    syscall::getpid().unwrap() as u32
```

Comment:
`pid_t` is a `usize` on redox, so this may lose bits.

Reply:
Does Redox really allow PIDs greater than 4 billion-ish though?

Reply:
If it does, we'll need to reconsider the current `Child::id` method.

Reply:
I made it a u32 to be consistent with `Child::id`.